modules/seqexec/server/src/main/scala/seqexec/server/SeqexecEngine.scala

```diff
@@ -277,7 +277,7 @@ class SeqexecEngine(httpClient: Client[IO], settings: Settings[IO], sm: SeqexecM
   def addSequencesToQueue(q: EventQueue, qid: QueueId, seqIds: List[Observation.Id])
   : IO[Either[SeqexecFailure, Unit]] = q.enqueue1(
     Event.modifyState[executeEngine.ConcreteTypes](addSeqs(qid, seqIds)
-      .map[executeEngine.ConcreteTypes#EventData](_ => UpdateQueueAdd(qid, seqIds)))
+      .as[executeEngine.ConcreteTypes#EventData](UpdateQueueAdd(qid, seqIds)))
```

Comment:
Why this change? Do we want to always evaluate `UpdateQueueAdd`? Or will that not happen?

Reply:
`UpdateQueueAdd` is an output event, that tells the client that a sequence was added to a queue.